test/parallel/forall/reduce-intents/ri-4-arrdom.chpl

```diff
@@ -25,13 +25,13 @@ proc setupDistributions() {
     return new Block(rank=2, boundingBox={1..d, 1..d});
 
   else if distType == DistType.cyclic then
-    return new dmap(new Cyclic(startIdx=(0,0)));
```

Comment:
Had this not been emitting a deprecation warning before?

Reply:
This test (and the next) weren't actually being tested.  I was just doing some git grepping of dmap + Cyclic | Replicated and ended up updating these tests despite their not being tested (and then kept going and updating other aspects of them that trivially weren't compiling).